backend/clap/proto/data_store.proto

```diff
@@ -182,6 +182,20 @@ message StudyVisitRequest {
     int32 submission_id = 2;
 }
 
+message EmailDetailsRequest {
+    int32 assessor_id         = 1;
+    int32 project_id          = 2;
+    bool is_after_finish_date = 3;
+}
+ 
+message EmailDetailsResponse {
+    string assessor_name        = 1;
+    string assessment_surname   = 2;
```

Comment:
`assessment_surname` do you mean `assessor_last_name` or `submission_name` ?
We should have both in the email either way